exercises/sum-of-multiples/canonical-data.json

```diff
@@ -1,11 +1,11 @@
 {
   "exercise": "sum-of-multiples",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "cases": [
     {
-      "description": "multiples of 3 or 5 up to 1",
+      "description": "multiples of 3 up to 1",
       "property": "sum",
-      "factors": [3, 5],
+      "factors": [3],
```

Comment:
Why do you want to change this test?
Why not add the new "simpler" test cases that you proposed?